# --- T2-COPYRIGHT-BEGIN ---
# t2/package/*/linux/amdkfd-riscv64.patch
# Copyright (C) 2025 The T2 SDE Project
# SPDX-License-Identifier: GPL-2.0 or patched project license
# --- T2-COPYRIGHT-END ---

--- linux-6.6/drivers/gpu/drm/amd/amdkfd/Kconfig.vanilla	2025-03-22 12:38:27.180581337 +0100
+++ linux-6.6/drivers/gpu/drm/amd/amdkfd/Kconfig	2025-03-22 12:38:55.703819058 +0100
@@ -5,7 +5,7 @@
 
 config HSA_AMD
 	bool "HSA kernel driver for AMD GPU devices"
-	depends on DRM_AMDGPU && (X86_64 || ARM64 || PPC64)
+	depends on DRM_AMDGPU && (X86_64 || ARM64 || PPC64 || (RISCV && 64BIT))
 	select HMM_MIRROR
 	select MMU_NOTIFIER
 	select DRM_AMDGPU_USERPTR
